e2e/content-services/share-file/share-file.e2e.ts

```diff
@@ -199,5 +201,19 @@ describe('Share file', () => {
             await BrowserActions.getUrl(sharedLink);
             await viewerPage.checkFileNameIsDisplayed(pngFileModel.name);
         });
+
+        it('[C260153] Should shared files listed in share files custom resources', async () => {
```

Comment:
this test name is not very clear

Reply:
what is the purpose of this file? to test the Share action from UI or to check that a shared file appears in the Shared Files list?

if the purpose is to test that a shared file appears in the Shared Files list, then the creating and sharing of the file would be a precondition, meaning the sharing of the file can be done using an API call, not using the UI.

Reply:
at this point is not anymore an e2e it becomes a unit test. E2E is to test a behavior flow this is the scope of it.  I see often this problem we write E2E thinking like a unit test. In this flow, we share a file and check if it appears in the list of the shared file.

 Should shared files listed in share files custom resources do you have a better name ?

Reply:
Maybe something like "Shared files are listed in the Shared custom source"

Reply:
OK, so from your comment above I understand you consider this an integration test, which in general are useful.

But somehow, for this specific case, I don't think it brings much value, cause:
- when you share a file from the UI, what basically happens is you set some node properties in the BE. This is covered by the other tests from this suite
- when you check the files are listed in the Shared files custom source, you're basically testing the API endpoint response, and for testing this, doesn't matter how the file is shared, if it's by a BE call or using the UI

Reply:
End-to-end testing is a methodology used to test whether the flow of an application is performing as designed from start to finish. The purpose of carrying out end-to-end tests is to identify system dependencies and to ensure that the right information is passed between various system components and systems

If you are testing only one behaviour of one component you can archive a better result with a unit test without spin up the environment backend and frontend. This is the reason why really few e2e make sense in ADF. This e2e was written and manual executed if you think it doesn't make sense as e2e we can deprecated but please not confuse e2e with unit test. If we etite e2e to test unit test scenario we are going only to have slower unit test instead of e2e